ui/app/components/clients/page/counts.hbs

```diff
@@ -12,8 +12,32 @@
     @upgradesDuringActivity={{this.upgradesDuringActivity}}
     @noData={{not @activity.total.clients}}
     @onChange={{this.onDateChange}}
+    @showEditModal={{this.showEditModal}}
+    @setEditModalVisible={{this.setEditModalVisible}}
   />
 </div>
+{{#if this.showCommunity}}
+  <div class="has-top-padding-m">
+    <Hds::Text::Display @tag="h3">Client counting period</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" class="has-text-grey">Specify a date range to view within that timeframe. Click 'Edit' to
+      choose a different date range.</Hds::Text::Body>
+    <Hds::Text::Body @tag="p">
+      <span data-test-date-range="start">{{this.formattedStartDate}}</span>
+      -
+      <span data-test-date-range="end">{{this.formattedEndDate}}</span>
+      <Hds::Button
+        @size="small"
+        @text="Edit"
+        @isInline="true"
+        @color="tertiary"
+        @icon="edit"
+        @iconPosition="trailing"
+        data-test-date-range-edit
+        {{on "click" (fn (mut this.showEditModal) true)}}
+      />
+    </Hds::Text::Body>
+  </div>
+{{/if}}
```

Comment:
Could this move into the `<Clients::PageHeader>` component since it's part of the header? That way we also wouldn't have to pass extra args `showEditModal` and `setEditModalVisible` all the way up for just this button.